app/_src/deck/reference/deck_file_namespace.md

```diff
@@ -0,0 +1,66 @@
+---
+title: deck file namespace
+source_url: https://github.com/Kong/deck/tree/main/cmd/file_namespace.go
+content_type: reference
+---
+
+Apply a namespace to routes in a decK file by prefixing the path.
+
+By prefixing paths with a specific segment, colliding paths to services can be
+namespaced to prevent collisions. For example, two API definitions that both expose a
+'/list' path. By prefixing one with '/addressbook' and the other with '/cookbook',
+the resulting paths '/addressbook/list' and '/cookbook/list' can be exposed without
+colliding.
```

Comment:
```suggestion
namespaced to prevent collisions. For example, two API definitions that both expose a
`/list` path. By prefixing one with `/addressbook` and the other with `/cookbook`,
the resulting paths `/addressbook/list` and `/cookbook/list` can be exposed without
colliding.
```